pkg/collector/autodiscovery/configresolver.go

```diff
@@ -267,17 +275,13 @@ func getContainerName(tplVar []byte, svc listeners.Service) []byte {
 }
 
 // getTags returns tags that are appended by default to all metrics.
-// TODO (use svc.Tags)
-func getTags(tplVar []byte, svc listeners.Service) []byte {
-	return []byte("[\"tag:foo\", \"tag:bar\"]")
-}
-
-// getOptTags returns tags that are to be applied to templates with %%tags%%.
-// This is generally reserved to high-cardinality tags that we want to provide,
-// but not by default.
-// TODO
-func getOptTags(tplVar []byte, svc listeners.Service) []byte {
-	return []byte("[\"opt:tag1\", \"opt:tag2\"]")
+func getTags(svc listeners.Service) []byte {
```

Comment:
We can safely remove this. Let's kill the `%%tags%%` tpl var as we add them anyway.

Reply:
Absolutely, I forgot to remove this 👍